### ISSUE TITLE
Title in Date and time dialog is not configurable

### ISSUE THREAD

**Issue by damiancioo** [vanpra/compose-material-dialogs#50]
**Problem**:
There is hardcoded 'SELECT TIME' in TimePicker.kt

**Possible solution**:
It should be possible to configure it similar to `colors: TimePickerColors`

**Comment by vanpra** [comment]
This is planned either for the next release or the one after it. I'm currently in the process of writing tests for all the modules which should be done soon so after I finish that I'll look into implementing this. See #37 for any further updates.

**Comment by damiancioo** [comment]
I just watching what is in 0.4.0, .... 
There is an ability to add (not change) title, so with code like:
```
): MaterialDialog = MaterialDialog().also {
    it.build {

        title(res = R.string.date_picker_select_time)

        timepicker(
            title = "Custom title",
```
There are both titles,  and second one is like in old version 'SELECT TIME', because:

```
@Composable
internal fun TimePickerTitle(text: String, onBack: (() -> Unit)?) {
    if (onBack != null) {
            Text(
                "SELECT TIME",
                style = TextStyle(color = MaterialTheme.colors.onBackground)
            )
    } else {
            Text(
                "SELECT TIME",
                modifier = Modifier.paddingFromBaseline(top = 28.dp),
                style = TextStyle(color = MaterialTheme.colors.onBackground)
            )
        }
}
```

Seriously! :angry:

**Comment by damiancioo** [comment]
Fixed in 0.4.1
